test/Test/Database/LSMTree/Internal/Run.hs

```diff
@@ -262,7 +262,7 @@ isLargeKOp (key, entry) = size > pageSize
 
 readKOps :: FS.HasFS IO h -> FS.HasBlockIO IO h -> Run RealWorld (FS.Handle h) -> IO [SerialisedKOp]
 readKOps fs hbio run = do
-    reader <- Reader.new fs hbio run
+    reader <- Reader.new fs hbio Nothing run
```

Comment:
Will we have tests for the new functionality later? :-) In this PR or another PR?

Reply:
Another PR I think. I'll need some guidance on the semantics of how to set up an "imperative" test correctly with our helper functions.

Reply:
There are tests for this now: https://github.com/IntersectMBO/lsm-tree/pull/346/files#diff-48a02884e73d81f478092b7cb891f5af973b8b6844257c98f597415d0652e159